### PR TITLE
Updates/harmonizing of specviz docs

### PR DESCRIPTION
**Pull request by eteq**
This PR is the output of a read-through of the specviz docs to try to give it a more unified voice/provide a bit more context.

I think it's all pretty straightforward, but the only thing I removed that might be controversial is the section that said what it might do "in the future".  I think it's wise to avoid that in documentation because the purpose of code-level documentation is to document what the tool is *right now*, not what it might be in the future.

### CODE REVIEW

**Comment by eteq** [comment]
(I may have gone a bit crazy with the highly granular commits, but :shrug:)

**Review by javerbukh — APPROVED** [review]
Looks good!

File: docs/specviz/import_data.rst [hunk 1]
```diff
@@ -33,10 +35,19 @@ Here, users can select the loaded data set to be visualized in the viewer.
...
+
+Loading data via the API
+------------------------
+Alternatively, if users are working in a coding environment like a Jupyter notebook, they have access to the :class:`~jdaviz.configs.specviz.helper.Specviz` helper class API. Using this API, users can load data into the application through code.
+Below is an example of importing the :class:`~jdaviz.configs.specviz.helper.Specviz` helper class, creating a :class:`~specutils.Spectrum1D` object from a data file via the :func:`~specutils.Spectrum1D.read` method::
+
+    >>> from specutils import Spectrum1D
+    >>> spec1d = Spectrum1D.read("/path/to/data/file") #doctest: +SKIP
+    >>> specviz = SpecViz() #doctest: +SKIP
+    >>> specviz.load_spectrum(spec1d)
```
Comment (javerbukh) [inline docs/specviz/import_data.rst:48]: Seems like this line is failing in the tests. Does Specviz need to be imported here?

Comment (eteq) [reply docs/specviz/import_data.rst:48]: Ah, thanks - no, it looks like I just forgot to add the `` #doctest: +SKIP`` on that line!

**Pull request merged**